examples/integration_test.py

```diff
@@ -1282,10 +1268,7 @@ def verify_avro_explicit_read_schema():
                 continue
 
             if msg.error():
-                if msg.error().code() == confluent_kafka.KafkaError._PARTITION_EOF:
-                    break
```

Comment:
this is a change in behaviour, it would previously break out on EOF. Now there is no exit condition in this loop.
Did this test really pass?

Reply:
Only ran the unit tests the first go to be honest. Fixed, running the full suite before pushing